### ISSUE TITLE
hi team-exor Can the function of Latest Blocks be added?

### ISSUE THREAD

**Issue by msy2008** [team-exor/eiquidus#14]
Can you add an option to switch to Display Latest Blocks on the homepage? Very useful for many cryptocurrencies, in fact it is not practical to display Latest Transactions, https://chainz.cryptoid.info/ also displays Latest Blocks, and I found a project that also displays Latest Blocks, but they do not provide source code.
![image](https://user-images.githubusercontent.com/9958239/163124717-95d9e5b3-8dae-4155-8e0d-71f5fa44fc2a.png)
![image](https://user-images.githubusercontent.com/9958239/163124958-5db05177-c71d-4859-b8fc-eba0f0250e52.png)

**Comment by joeuhren** [comment]
I agree it would be useful to display blocks instead of transactions on the main page, but the changes would take a bit of effort and require a full resync to get the data into the newer format. For now, eiquidus is aiming to be a continuation of the original iquidus block explorer which sorts the data into transactions instead of blocks.

In the future, I may change this to be split out separately so there can be a page for blocks and a page for transactions if desired, but I have other priorities at this time.

If you or anyone out there reading this is interested in adding this feature, I would gladly accept a well thought out pull request that can accomplish this.

**Comment by joeuhren** [comment]
Please note that this request has been added to Exor's public task list and development will begin as soon as the funding goal has been met.

View more details here: https://exor.io/task?id=3&tid=ca52af09c74046251bd2ba4481dce3362a07d616
